.circleci/config.yml

```
@@ -332,7 +332,7 @@ jobs:
               echo "export TEST_MARKERS='<< parameters.package_type >> and upgrade'" >> $BASH_ENV
           fi
       - run_pytest:
-          pytest_options: "-n auto -m \"$TEST_MARKERS\" --test-bundle-path ./signalfx-agent-latest.tar.gz"
+          pytest_options: "-n auto -m \"$TEST_MARKERS\" --test-bundle-path=\"./signalfx-agent-latest.tar.gz\""
```

Mostly to play with the new feature but there's also more quoting than necessary I think:
```suggestion
          pytest_options: -n auto -m "$TEST_MARKERS" --test-bundle-path=./signalfx-agent-latest.tar.gz
```